system-x/services/jms/rabbitmq/src/main/java/software/tnb/jms/rabbitmq/account/RabbitmqAccount.java

```diff
@@ -0,0 +1,24 @@
+package software.tnb.jms.rabbitmq.account;
+
+import software.tnb.common.account.Account;
+
+public class RabbitmqAccount implements Account {
+    private String username = "guest";
+    private String password = "guest";
```

Comment:
can you please create longest password (at least 16 chars), to avoid possible errors on FIPS-enabled environment please? [solution](https://access.redhat.com/solutions/6975925), [discussion](https://bugzilla.redhat.com/show_bug.cgi?id=2007331)